flask_restful/__init__.py

```diff
@@ -301,12 +301,7 @@ def handle_error(self, e):
         headers = {}
 
         if code >= 500:
-            # There's currently a bug in Python3 that disallows calling
```

Comment:
I like the idea of using `app.log_exception`, but I think you went a bit too far with your refactoring. This comment was here to warn about [stuff like this](https://travis-ci.org/flask-restful/flask-restful/builds/78217023)